### DISCUSSION TITLE
How do you sign-in on macOS with an Apple ID?

### DISCUSSION THREAD

**Discussion by akhil-hari** [quickemu-project/quickemu#1253] [Quickemu]
Cant sign into apple id on mac os big sur

**Comment by Munir-Hassan** [comment]
> Cant sign into apple id on mac os big sur

@akhil-hari Hi, Did you find a way to login to apple id in macOS? I need to download and install xcode.

**Comment by salahdabbech1** [comment]
hey, so following the readme there is a this [repo](https://github.com/quickemu-project/qe_mac_apid.git) developed by the team, supposed to generate a random but realistic looking MAC address and and a UUID.
Now the issue is that whenever i try to use it the check coverage website it would return the following error message: 

> please enter a valid serial number

different than we are supposed to be getting. 
Another thing, should we check the coverage of the random mac serial number in the same place i set my mac's location at setup?
